odl/solvers/iterative/statistical.py

```diff
@@ -104,34 +96,33 @@ def osmlem(op, x, data, niter, noise='poisson', callback=None, **kwargs):
         Vector to which the result is written. Its initial value is
         used as starting point of the iteration, and its values are
         updated in each iteration step.
-    data : sequence of ``op.range`` `element-like`
+        The initial value of ``x`` should be non-negative.
+      data : sequence of ``op.range`` `element-like`
         Right-hand sides of the equation defining the inverse problem.
     niter : int
         Number of iterations.
-    noise : {'poisson'}, optional
-        Noise model determining the variant of MLEM.
-        For ``'poisson'``, the initial value of ``x`` should be
-        non-negative.
     callback : callable, optional
         Function called with the current iterate after each iteration.
 
     Other Parameters
     ----------------
     sensitivities : float or ``op.domain`` `element-like`, optional
-        Usable with ``noise='poisson'``. The algorithm contains an ``A^T 1``
+        The algorithm contains an ``A^T 1``
         term, if this parameter is given, it is replaced by it.
         Default: ``op[i].adjoint(op[i].range.one())``
 
     Notes
     -----
-    Given a forward models :math:`A_i`, data :math:`g_i`, :math:`i = 1, ..., M`
-    and noise model :math:`X`, the algorithm attempts find an :math:`x` that
+    Given a forward models :math:`A_i`, and data :math:`g_i`,
```

Comment:
models -> model

Reply:
I now wrote "model", but the original intent might have been "Given forward models A_i"? Doesn't matter much.

Reply:
Oh yes, that was probably the intention. I didn't read beyond the first part :-)

Reply:
Ok, it's `models` now.